language/diem-framework/modules/DiemAccount.move

```diff
@@ -1400,12 +1397,15 @@ module DiemFramework::DiemAccount {
         DualAttestation::publish_credential(&new_account, creator_account, human_name);
         VASPDomain::publish_vasp_domains(&new_account);
         make_account(&new_account, auth_key_prefix);
-        add_currencies_for_account<Token>(&new_account, add_all_currencies)
+        add_currencies_for_account<Token>(&new_account, add_all_currencies);
+        spec {
+            assert exists<VASPDomain::VASPDomains>(Signer::address_of(new_account));
+            assert Roles::spec_has_treasury_compliance_role_addr(Signer::address_of(creator_account));
+        }
```

Comment:
these inline spec is at the end of the function, and my reading is that these inline spec should already cover by either an `ensures` or an `aborts_if`